apps/bridge/src/components/DeprecationContent/BridgeCards.tsx

```diff
@@ -45,14 +45,17 @@ const bridges = coinflip([
 
 function BridgeCard({ name, url, logo, color, team }: (typeof bridges)[0]) {
```

Comment:
I wonder if it might make sense to make these cards be a bit more generic and reusable in other places in the website.

Reply:
We definitely need a card component for base.org, now the bridge is just a stub of a site